src/utils/WalletTxsProvider.js

```diff
@@ -0,0 +1,40 @@
+import { AbstractBlockDataProvider } from './BlockDataPaginator'
+import TokenModel from '../models/TokenModel'
+import ERC20DAO from '../dao/ERC20DAO'
+import TransactionModel from '../models/TransactionModel'
+import Immutalbe from 'immutable'
```

Comment:
typo Immuta**l**be
**Use TYPO checker in your WebStorm and never commit code with typos!**